docs/get_started/get_started.md

```diff
@@ -13,7 +13,7 @@ To install Cerebral BETA you have to explicitly install Cerebral and its depende
 
```

Comment:
Above Cerebral is introduced as beta.

Reply:
These docs are not used, should be deleted :)